tests/queries/0_stateless/00965_shard_unresolvable_addresses.sql

```diff
@@ -1,2 +1,2 @@
-SELECT count() FROM remote('127.0.0.1,localhos', system.one); -- { serverError 279 }
+SELECT count() FROM remote('127.0.0.1,localhos', system.one); -- { serverError 198 }
```

Comment:
I think that DNS_ERROR (198) is more informative here that ALL_CONNECTION_TRIES_FAILED (279), so made this change in test.